### PR TITLE
feat: adicionando rotas e criando base para widget de página de registro

### PR DESCRIPTION
**Pull request by Thentrax**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -37,7 +37,11 @@ class MyApp extends StatelessWidget {
           brightness: Brightness.dark,
         ),
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      initialRoute: '/',
+      routes: {
+        '/': (context) => const MyHomePage(title: 'SpotOn'),
+        '/register': (context) => Container(),
```
Comment (biel-correa) [inline lib/main.dart:43]: Extract routes names to a class to keep everything centralized

Comment (TiFurt) [reply lib/main.dart:43]: extract routes and name. Like this example:
image

File: lib/main.dart [hunk 2]
```diff
@@ -38,7 +39,8 @@ class MyApp extends StatelessWidget {
           brightness: Brightness.dark,
         ),
       ),
-      home: const PresentationPage(),
+      routes: routes,
+      initialRoute: '/',
```
Comment (biel-correa) [inline lib/main.dart:43]: Use RouteNames.home instead

File: lib/pages/presentation_page/presentation_page.dart [hunk 3]
```diff
@@ -56,7 +56,7 @@ class PresentationPage extends StatelessWidget {
     print('Navigate to login');
   }
 
-  void _navigateToRegister() {
-    print('Navigate to register');
+  void _navigateToRegister(context) {
+    Navigator.pushNamed(context, '/register');
```
Comment (biel-correa) [inline lib/pages/presentation_page/presentation_page.dart:60]: Use RouteNames.register instead

**Review by TiFurt — APPROVED** [review]

**Review by biel-correa — APPROVED** [review]
LGTM

**Pull request merged**